app.py

```diff
@@ -364,6 +364,9 @@ def settlement():
     if not bank or bank.user != api_key.user:
         return bad_request("invalid bank account")
     merchant_rate = api_key.user.merchant_rate if api_key.user.merchant_rate else app.config["MERCHANT_RATE"]
+    merchant_name = api_key.user.merchant_name 
+    if not merchant_name: 
+        return bad_request("blank merchant name")
```

Comment:
can we change this message to: "merchant name not set"

Reply:
changed the message.